client/src/interface/Profile.ts

```diff
@@ -11,6 +12,14 @@ export interface Profile {
   photo: string;
 }
 
+export interface PetSitter extends Profile {
```

Comment:
Hey! would you please explain the use of this interface as far filtering the navbar links is concerned

Reply:
Now! Now I get! Instead of minding about which kind of data your getting from the back! You will have all the properties once in PetSitter interface

Reply:
💯